packages/apollo/src/commands/service/check.ts

```diff
@@ -97,6 +116,59 @@ ${
 `;
 }
 
+export function formatHumanReadable({
+  checkSchemaResult
+}: {
+  checkSchemaResult: CheckSchema_service_checkSchema;
+}): string {
```

Comment:
Yes, we can let TS infer the return type. I defined it as a check to myself to make sure I always return a string and not something else (even if it's inferred) by accident. If it's a common practice to infer the return type then we should stick with it.

Reply:
Totally ok with leaving as-is! My default has always been to let it infer unless we need to tell it otherwise, but I appreciate your reason and am reconsidering my default.